website/frontend/src/components/blocks/UploadForm/children/MetaFile/MetaFile.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { FC, useRef } from 'react';
+import { useDispatch } from 'react-redux';
+import { Field } from 'formik';
+import clsx from 'clsx';
+import { Trash2 } from 'react-feather';
+import { AddAlert } from 'store/actions/actions';
+import { checkFileFormat } from 'helpers';
+import { EventTypes } from 'types/alerts';
+import styles from './MetaFile.module.scss';
+
+type Props = {
+  droppedMetaFile: File | null;
+  handleUploadMetaFile: (file: File) => void;
+  handleRemoveMetaFile: () => void;
+};
+
+export const MetaFile: FC<Props> = ({ droppedMetaFile, handleUploadMetaFile, handleRemoveMetaFile }) => {
+  const dispatch = useDispatch();
+  const metaFieldRef = useRef<HTMLDivElement | null>(null);
```

Comment:
```suggestion
  const metaFieldRef = useRef<HTMLDivElement>(null);
```

You can omit `| null` since it's read only ref